libs/vkd3d/command.c

```diff
@@ -8822,6 +8887,8 @@ static HRESULT d3d12_command_list_init(struct d3d12_command_list *list, struct d
 
     list->type = type;
 
+    list->command_list_vkd3d_ext.ID3D12GraphicsCommandListExt_iface.lpVtbl = &d3d12_command_list_vkd3d_ext_vtbl;
```

Comment:
It feels somewhat unnecessary to have a separate struct here which requires chaining two CONTAINING_RECORDs. `list->ID3D12GraphicsCommandListExt_iface` seems sensible to me.

Reply:
hmm, yeah looks like currently we do not have any other member to add, makes sense to remove redundant chaining. Will do.

Reply:
same goes for the device.